docs/Exp04-Tutor.md

```diff
@@ -56,17 +56,96 @@ Now, every time you open a new terminal, you can initialize the environment of B
 BLIS_INIT
 ```
 
-### 2.3. Intel CPU
+### 2.3. MKL-Intel CPU
 
-To do.
+#### 2.3.1 Intel oneAPI Base Toolkit
+
+If you have an Intel CPU, you should install **MKL** for BLAS.
+
+It's included by **Intel oneAPI Base Toolkit**. So we'll install the toolkit. Here is the official download [page](https://www.intel.cn/content/www/cn/zh/developer/tools/oneapi/base-toolkit-download.html).
+
+Choose the `Offline/Online Installer` for any system. If your **Package Manager**'s package type is supported, you can choose it instead.
+
+The website and installer (if you choose installer) will guide you, just follow them.
+
+After installation, find `setvar.sh`. Usually, you'll find it in `~/intel/oneapi/setvars.sh` or `/opt/intel/oneapi/setvars.sh`. Source it with command:
+
+
+```bash
+# choose one to source
+source ~/intel/oneapi/setvars.sh
+# or if it's in /opt
+source /opt/intel/oneapi/setvars.sh
+```
+
+If you see outputs like:
+
+```bash
+ 
+:: initializing oneAPI environment ...
+   bash: BASH_VERSION = 5.1.16(1)-release
+   args: Using "$@" for setvars.sh arguments: 
+:: advisor -- latest
+:: ccl -- latest
+:: compiler -- latest
+:: dal -- latest
+:: debugger -- latest
+:: dev-utilities -- latest
+:: dnnl -- latest
+:: dpcpp-ct -- latest
+:: dpl -- latest
+:: ipp -- latest
+:: ippcp -- latest
+:: mkl -- latest
+:: mpi -- latest
+:: tbb -- latest
+:: vtune -- latest
+:: oneAPI environment initialized ::
+ 
+```
+
+It means the initialization and the installation are successful.
+
+You must source the `setvar.sh` to initialize the environment before using Intel toolkit. It's troublesome to type such a long command, so let's set alias.
+
+Add the following line to the end of file `/etc/bash.bashrc` or `~/.bashrc`(Any of them is ok. They're both profile for bash)
+
+```bash
+# source intel vars
+alias INTEL_INIT='source ~/intel/oneapi/setvars.sh'
+# or if your intel home is in /opt
+alias INTEL_INIT='source /opt/intel/oneapi/setvars.sh'
```

Comment:
I personally think `ONEAPI_INIT` is a better alias, to avoid ambiguity with other Intel packages, if any.